apps/web/components/eventtype/SkeletonLoader.tsx

```diff
@@ -5,13 +5,13 @@ function SkeletonLoader() {
   return (
     <SkeletonContainer>
       <div className="mb-4 flex items-center">
-        <SkeletonAvatar className="h-8 w-8" />
+        <SkeletonAvatar width="8" height="8" />
```

Comment:
If im not mistaken this is bad practise and we should be using the w/h method in classNames - @zomars recently made a change to support this

Reply:
It's ok since this is a v1 view

Reply:
Will be deprecated anyways